api/Blob.json

```diff
@@ -208,9 +208,16 @@
                 "prefix": "webkit"
               }
             ],
-            "chrome_android": {
-              "version_added": true
-            },
+            "chrome_android": [
+              {
+                "version_added": "25"
+              },
+              {
+                "version_added": "5",
```

Comment:
Whoops, looks like this got set as “5” when it should be “18”, correct?

Reply:
Sorry. Fixed.

Reply:
All good, thanks for the quick fixes! This pull request looks all ready to go!